pkg/storage/unified/search/bleve_mappings.go

```diff
@@ -17,6 +17,12 @@ func getBleveMappings(fields resource.SearchableDocumentFields) mapping.IndexMap
 func getBleveDocMappings(_ resource.SearchableDocumentFields) *mapping.DocumentMapping {
 	mapper := bleve.NewDocumentStaticMapping()
 
+	nameMapping := &mapping.FieldMapping{
+		Type:  "text",
+		Index: true,
```

Comment:
this will need the keyword tokenizer (eg raw value)

Not important yet, but I think we will want DocValues, but not stored